tests/ui-tests/admin-portal/src/test/java/org/wso2/is/portal/admin/test/ui/bean/AdminPortalURLBean.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.is.portal.admin.test.ui.bean;
+
+/**
+ * This is the bean class to retrieve different urls/pages of the admin-portal
+ */
+public class AdminPortalURLBean {
+
+    private static String adminLoginPage = "https://" + System.getProperty("home") + ":" +
```

Comment:
Won't this make the test fail if the context path 'admin-portal' is changes to something else.